inobounce.js

```diff
@@ -22,6 +22,11 @@
 				break;
 			}
 
+            //ignore range input element
+            if (el.nodeName === 'INPUT' && el.hasAttribute('type') && el.getAttribute('type') === 'range') {
```

Comment:
We can safely call `getAttribute` without checking `hasAttribute` first, it will return `null`. I'd remove `hasAttribute` to avoid another function call.